src/metabase/pulse/render/table.clj

```diff
@@ -144,16 +144,16 @@
   background color for a given cell. `column-names` is different from the header in `header+rows` as the header is the
   display_name (i.e. human friendly. `header+rows` includes the text contents of the table we're about ready to
   create. If `normalized-zero` is set (defaults to 0), render values less than it as negative"
-  ([color-selector column-names contents]
-   (render-table color-selector 0 column-names contents))
+  ([color-selector column-names-map contents]
+   (render-table color-selector 0 column-names-map contents))
 
-  ([color-selector normalized-zero column-names [header & rows]]
+  ([color-selector normalized-zero {:keys [col-names cols-for-color-lookup]} [header & rows]]
```

Comment:
here we pass the column names for display as `col-names` and the original column names for the color lookup. This fixes the conditional formatting not showing up at all.